test/e2e/kibishii_utils.go

```diff
@@ -33,10 +33,59 @@ const (
+func runKibishiiTests(client testClient, providerName, veleroCLI, veleroNamespace, backupName, restoreName, backupLocation string,
+	useVolumeSnapshots bool, registryCredentialFile string) error {
+	oneHourTimeout, _ := context.WithTimeout(context.Background(), time.Minute*60)
+
+	if err := createNamespace(oneHourTimeout, client, kibishiiNamespace); err != nil {
+		return errors.Wrapf(err, "Failed to create namespace %s to install Kibishii workload", kibishiiNamespace)
+	}
+	defer func() {
+		if err := deleteNamespace(oneHourTimeout, client, kibishiiNamespace, true); err != nil {
+			fmt.Println(errors.Wrapf(err, "failed to delete the namespace %q", kibishiiNamespace))
+		}
+	}()
+	if err := kibishiiPrepareBeforeBackup(oneHourTimeout, client, providerName, kibishiiNamespace, registryCredentialFile); err != nil {
+		return errors.Wrapf(err, "Failed to install and prepare data for kibishii %s", kibishiiNamespace)
+	}
+
+	if err := veleroBackupNamespace(oneHourTimeout, veleroCLI, veleroNamespace, backupName, kibishiiNamespace, backupLocation, useVolumeSnapshots); err != nil {
+		veleroBackupLogs(oneHourTimeout, veleroCLI, veleroNamespace, backupName)
+		return errors.Wrapf(err, "Failed to backup kibishii namespace %s", kibishiiNamespace)
+	}
+
+	if providerName == "vsphere" && useVolumeSnapshots {
+		// Wait for uploads started by the Velero Plug-in for vSphere to complete
+		// TODO - remove after upload progress monitoring is implemented
+		fmt.Println("Waiting for vSphere uploads to complete")
+		if err := waitForVSphereUploadCompletion(oneHourTimeout, time.Hour, kibishiiNamespace); err != nil {
+			return errors.Wrapf(err, "Error waiting for uploads to complete")
+		}
+	}
+	fmt.Printf("Simulating a disaster by removing namespace %s\n", kibishiiNamespace)
+	if err := deleteNamespace(oneHourTimeout, client, kibishiiNamespace, true); err != nil {
+		return errors.Wrapf(err, "failed to delete namespace %s", kibishiiNamespace)
+	}
+
+	if err := veleroRestore(oneHourTimeout, veleroCLI, veleroNamespace, restoreName, backupName); err != nil {
+		veleroRestoreLogs(oneHourTimeout, veleroCLI, veleroNamespace, restoreName)
+		return errors.Wrapf(err, "Restore %s failed from backup %s", restoreName, backupName)
+	}
+
+	if err := kibishiiVerifyAfterRestore(client, kibishiiNamespace, oneHourTimeout); err != nil {
+		return errors.Wrapf(err, "Error verifying kibishii after restore")
+	}
+
+	fmt.Printf("kibishii test completed successfully\n")
+	return nil
+}
+
 func installKibishii(ctx context.Context, namespace string, cloudPlatform string) error {
 	// We use kustomize to generate YAML for Kibishii from the checked-in yaml directories
+	//debug:vmware-tanzu-experiments
 	kibishiiInstallCmd := exec.CommandContext(ctx, "kubectl", "apply", "-n", namespace, "-k",
-		"github.com/vmware-tanzu-experiments/distributed-data-generator/kubernetes/yaml/"+cloudPlatform)
+		"github.com/danfengliu/distributed-data-generator/kubernetes/yaml/"+cloudPlatform)
```

Comment:
Can we revert this now that https://github.com/vmware-tanzu-experiments/distributed-data-generator/pull/9 has been merged?

Reply:
sorry for that, it's reverted.